tests/Component/RequestBuilder/FacetSetTest.php

```diff
@@ -288,6 +288,25 @@ public function testBuildWithFacetsAndGlobalFacetSettings()
         $this->component->addFacet(
             new FacetMultiQuery(['key' => 'f3', 'query' => ['f4' => ['query' => 'category:40']]])
         );
+        $request = $this->builder->buildComponent($this->component, $this->request);
+        static::assertEquals(
+            null,
+            $request->getRawData()
+        );
+        static::assertEquals(
+            '?facet.field={!key=f1}owner&facet.query={!key=f2}category:23&facet.query={!key=f4}category:40&facet=true&facet.missing=true&facet.limit=10',
+            urldecode($request->getUri())
+        );
+    }
+
+    public function testBuildWithFacetsAndSameFieldMultiplePrefix()
+    {
+        $this->component->setMissing(true);
+        $this->component->setLimit(10);
+        $this->component->addFacet(new FacetField(['key' => 'f1', 'field' => 'owner']))->setPrefix('Y');
```

Comment:
You set the Prefix on the FacetSet. Wouldn't it be correct to set the Prefix on the Facet?
```
$this->component->addFacet(new FacetField(['key' => 'f1', 'field' => 'owner'])->setPrefix('Y'));
```

Reply:
Ups, wrong bracket counting...